### ISSUE TITLE
[2.0.0-alpha] Conflicting library definitions for 'node' found

### ISSUE THREAD

**Issue by frogcjn** [Microsoft/TypeScript#9494]
when using the new way to install d.ts introduced by https://github.com/Microsoft/TypeScript/issues/9184 

```
 npm install @types/express --save
 npm install @types/mongoose --save
```
then there is a conflict:

```shell
[100;30m6 /// <reference types="node" />
[100;30m  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@types/mongoose/index.d.ts(6,1): message TS4090: Conflicting library definitions for 'node' found at 'node_modules/@types/mongoose/node_modules/@types/node/index.d.ts' and 'node_modules/@types/express-serve-static-core/node_modules/@types/node/index.d.ts'. Copy the correct file to the 'typings' folder to resolve this conflict.
[100;30m6 /// <reference types="node" />
[100;30m  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@types/serve-static/node_modules/@types/express-serve-static-core/index.d.ts(6,1): message TS4090: Conflicting library definitions for 'node' found at 'node_modules/@types/serve-static/node_modules/@types/node/index.d.ts' and 'node_modules/@types/express-serve-static-core/node_modules/@types/node/index.d.ts'. Copy the correct file to the 'typings' folder to resolve this conflict.
```

**Comment by frogcjn** [comment]
tough `"skipLibCheck": true` in `tsconfig.json`, there is still error log from tsc.

**Comment by weswigham** [comment]
What version of npm are you running, and what other `@types` packages do you have installed?

**Comment by mhegazy** [comment]
The first line in express\index.d.ts is:

```ts
import * as serveStatic from "../serve-static";
```

and should be changed to:

```ts
/// <reference types="serve-static" />

import * as serveStatic from "serve-static";
```

and the package needs to have a dependency on `@types\server-static`

**Comment by frogcjn** [comment]
@weswigham 
npm version 3.10.2

└─┬ @types/mongoose@3.8.25-alpha 
  └── @types/node@4.0.24-alpha 


└── @types/express@4.0.26-alpha

**Comment by mhegazy** [comment]
should be fixed in `@types/express@4.0.28-alpha`
